tests/testutils/ctx.go

```diff
@@ -174,7 +174,12 @@ func (ctx *RktRunCtx) RunGC() {
 		"gc",
 		"--grace-period=0s",
 	)
-	if err := exec.Command(ctx.rktBin(), rktArgs...).Run(); err != nil {
+
+	cmd := exec.Command(ctx.rktBin(), rktArgs...)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stdout
```

Comment:
On semaphore, stdout/stderr is all I can grab, so this change is necessary for debugging (especially when I can't reproduce failing test in my local machine). Also, the change only affects the functional tests. The question is how to log it somewhere else?